esp/templates/program/modules/teacheracknowledgementmodule/acknowledgement.html

```diff
@@ -1,11 +1,12 @@
 {% extends "main.html" %}
+{% load modules %}
 {% block title %}{{program.niceName}} Teacher Acknowledgement{% endblock %}
 {% block content %}
-<h2>Teacher Acknowledgement for {{program.niceName}} </h2>
+<h2>Teacher{% if prog|hasModule:"TeacherModeratorModule" %}/{{ prog.getModeratorTitle }}{% endif %} Acknowledgement for {{program.niceName}} </h2>
 <br />
 {% load render_qsd %}
 {% inline_program_qsd_block program "teach:acknowledgement_instructions" %}
-{{ prog.niceName }} will be on {{ prog.date_range }}! Please acknowledge that you are aware of this and commit to teaching at one of the times you have specified for your availability. You may not cancel your classes after student registration opens. In the case of extenuating circumstances like illness or serious emergency, you must notify the directors ASAP at <a href="mailto:{{ prog.director_email }}">{{ prog.director_email }}</a> and assist in finding a substitute for your classes. If you do not meet these expectations, it is unfair for the students who registered for your class and makes organization difficult for us. Note that not being responsible with respect to your classes will affect your ability to teach for us in the future.
+{{ prog.niceName }} will be on {{ prog.date_range }}! Please acknowledge that you are aware of this and commit to being a teacher{% if prog|hasModule:"TeacherModeratorModule" %} and/or {{ prog.getModeratorTitle|lower }}{% endif %} for one or more of the times you have specified for your availability. Classes may not be cancelled after student registration opens. In the case of extenuating circumstances like illness or serious emergency, you must notify the directors ASAP at <a href="mailto:{{ prog.director_email }}">{{ prog.director_email }}</a> and assist in finding a substitute for your classes. If you do not meet these expectations, it is unfair for the students who registered for your class and makes organization difficult for us. Note that not being responsible with respect to your classes may affect your ability to be a teacher{% if prog|hasModule:"TeacherModeratorModule" %} and/or {{ prog.getModeratorTitle|lower }}{% endif %} with us in the future.
```

Comment:
This looks good on dev3, but the checkbox at the bottom of the form needs to also reference moderating if moderator module:

![image](https://user-images.githubusercontent.com/25392075/114794038-a755cb80-9d59-11eb-83ba-b8160018bd00.png)